Services/data/devicemodels/scripts/TempIncrease-method.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/*global log*/
+/*global updateState*/
+/*global sleep*/
+/*jslint node: true*/
+/*jslint todo: true*/
+
+"use strict";
+
+/*jslint unparam: true*/
+function main(context, previousState) {
```

Comment:
To match the latest js file format, should be: `function main(context, previousState, previousProperties) {`

Reply:
Also consider adding comments to match the other files:
```
/**
 * Restore the global state using data from the previous iteration.
*
* @param previousState device state from the previous iteration
* @param previousProperties device properties from the previous iteration
*/
```

Reply:
Yeah i started with adding previousProperties and then realized they are not really needed for method execution. I thought it would be better to not include. Let me know if these are going to be needed for method execution.